test/src/examples_backend_test.go

```diff
@@ -0,0 +1,36 @@
+package test
+
+import (
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+// Test the Terraform module in examples/backend using Terratest.
+func TestExamplesBackend(t *testing.T) {
+	t.Parallel()
```

Comment:
Why have `t.Parallel()` if you are going to run the tests all sequentially? Either remove `t.Parallel()` or copy [code from example](https://github.com/cloudposse/terraform-example-module/blob/2b51aedca9fc38f156aec6818d74cd2fbc5101f6/test/src/examples_complete_test.go#L14-L43) to run them in parallel and change the Makefile.

Reply:
updated